include/cutest.h

```diff
@@ -526,9 +528,9 @@ main(int argc, char** argv)
     test_argv0__ = argv[0];
 
 #if defined CUTEST_UNIX__
-    test_colorize__ = isatty(fileno(stdout));
+    test_colorize__ = isatty(STDOUT_FILENO);
 #elif defined CUTEST_WIN__
-    test_colorize__ = _isatty(_fileno(stdout));
+    test_colorize__ = _isatty(STDOUT_FILENO);
```

Comment:
For Windows, `_isatty(_fileno(stdout))` should be kept. Windows SDK does not have `<unistd.h>`